src/__tests__/Search.test.tsx

```diff
@@ -86,21 +86,28 @@ describe("Renders Search Results Page", () => {
     test("clicking 'filters' button shows filters contents", () => {
       fireEvent.click(screen.getByText("Filters (0)"));
       expect(
-        screen.getByRole("combobox", { name: "Items Per Page" })
+        screen.getAllByRole("combobox", { name: "Items Per Page" })[0]
```

Comment:
why is there more than one combobox with "items per page"?

Reply:
Due to how Chakra loads responsive styling, the elements were accessible through the tests. I changed how the display property is populated by setting a state for the breakpoint width (this is how it is done on design system).